openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -23,13 +23,7 @@
     <header class="page-header has-secondary">
         <div class="page-header-main">
             <nav aria-label="${_('Course Outline')}" class="sr-is-focusable" tabindex="-1">
-                % if waffle.flag_is_active(request, UNIFIED_COURSE_EXPERIENCE_FLAG):
-                    <h2 class="hd hd-3 page-title">
-                        ${_("{course_name}").format(org=course.display_org_with_default, course_name=course.display_name_with_default)}
```

Comment:
That was just a poor implementation from earlier that I noticed while I was in here. This was originally a complicated format string, but in the end we decided to use just the course_name. There was no need for localization and interpolation to just use the name!